DFW.h

```diff
@@ -1,18 +1,33 @@
-// Serial controller library for DFWireless controllers
+/**
+ * Serial controller library for DFWireless controllers
+ * For RBE Class use
+ * @author Joe St. Germain joest@wpi.edu
+ * @author Alex Taglieri (atags22)
+ * @author Kevin Harrington (madhephaestus)

```

Comment:
Commit history is good too, except Joes contributions would be obliterated. I want to keep the author tags wherever possible.